File: kevoree-library/frascati/org.kevoree.library.frascati.mavenplugin/src/main/scala/org/kevoree/library/frascati/mavenplugin/Tester.scala
package org.kevoree.library.frascati.mavenplugin

import java.io.File
import org.kevoree.framework.KevoreeXmiHelper

/**
 * Created by IntelliJ IDEA.
 * User: duke
 * Date: 30/01/12
 * Time: 17:44
 */

object Tester extends App {

  KevoreeXmiHelper.save("/tmp/test.kev",CompositeParser.parseCompositeFile(new File("/opt/frascati-runtime-1.4/examples/helloworld-pojo/src/main/resources/helloworld-pojo.composite"),"1.5.1-SNAPSHOT","org.kevoree.library.javase","org.kevoree.library.javase.test","helloworld-pojo.composite"))
}

File: kevoree-platform/org.kevoree.platform.base/src/main/java/org/kevoree/platform/standalone/Tester.java
package org.kevoree.platform.standalone;

import org.kevoree.kcl.KevoreeJarClassLoader;

/**
 * Created with IntelliJ IDEA.
 * User: duke
 * Date: 1/28/13
 * Time: 9:05 AM
 */
public class Tester {

    public static void main(String[] args){
        KevoreeJarClassLoader jar = new KevoreeJarClassLoader();
        Class kevRoot = jar.loadClass("org.kevoree.ContainerRoot");
        Class tuple2 = jar.loadClass("jet.Tuple2");
    }




}
